bucket/glimpse.json

```diff
@@ -0,0 +1,76 @@
+{
+    "version": "0.2.0",
+    "description": "Glimpse is an open source image editor based on the GNU Image Manipulation Program. The goal is to experiment with new ideas and expand the use of free software.",
+    "homepage": "https://glimpse-editor.github.io/",
+    "license": "GPL-3.0-or-later",
+    "architecture": {
+        "64bit": {
+            "extract_dir": "PFiles\\Glimpse Image Editor\\Glimpse 0.2.0 (64-bit)",
```

Comment:
Move this property after hash property

Reply:
Is there a linter or fixer script that can be ran while writing or editing manifests before submission?